dataclients/kubernetes/testdata/ingressV1/ingress-data/ing-with-duplicate-host-different-service.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  namespace: foo
+  name: qux
+spec:
+  defaultBackend:
+    service:
+      name: bar
+      port:
+        number: 1234
+  rules:
+  - host: www.example.org
+    http:
+      paths:
+      - path: "/"
+        pathType: ImplementationSpecific
+        backend:
+          service:
+            name: bar
+            port:
+              name: baz
+  - host: www.example.org
+    http:
+      paths:
+      - path: "/"
+        pathType: ImplementationSpecific
+        backend:
+          service:
+            name: bar
+            port:
+              name: baz
```

Comment:
Service seems to be the same though - `bar`

Reply:
Should be fixed I guess?

Reply:
yes, will fix it. Just rebased to remove go.mod changes